metadata-ingestion/src/datahub/ingestion/source/sql/postgres.py

```diff
@@ -144,16 +161,12 @@ def create(cls, config_dict, ctx):
         return cls(config, ctx)
 
     def get_inspectors(self) -> Iterable[Inspector]:
-        # This method can be overridden in the case that you want to dynamically
-        # run on multiple databases.
         url = self.config.get_sql_alchemy_url()
         logger.debug(f"sql_alchemy_url={url}")
+        # Set default database to `postgres` over postgres default of the user's username.
```

Comment:
This comment seems out of place here because the connection will use the `database` field in `**self.config.options`, which might not be `postgres` anymore if the user explicitly set it.